DQM/L1TMonitor/python/L1TStage2Emulator_cff.py

```diff
@@ -42,12 +42,14 @@
 )
 
 # OMTF
-from L1Trigger.L1TMuonOverlap.simOmtfDigis_cfi import *
+from L1Trigger.L1TMuonOverlapPhase1.simOmtfDigis_cfi import *
 valOmtfDigis = simOmtfDigis.clone(
     srcDTPh = "omtfStage2Digis",
     srcDTTh = "omtfStage2Digis",
     srcCSC = "omtfStage2Digis",
-    srcRPC = "omtfStage2Digis"
+    srcRPC = "omtfStage2Digis",
+    bxMin = cms.int32(-3),
+    bxMax = cms.int32(4)
```

Comment:
Please avoid type declarations in cloned modules as recommended here:
Point 17 in http://cms-sw.github.io/cms_coding_rules.html#6--packaging-rules-1

Reply:
So do you mean it should be 
    bxMin = -3,
    bxMax = 4
in the DQM/L1TMonitor/python/L1TStage2Emulator_cff.py?

This will require adding default values of the bxMin and bxMax to the L1Trigger/L1TMuonOverlapPhase1/python/simOmtfDigis_cfi.py (which in principle is not a problem)?

Reply:
Yes, that is what I mean, and yes please add them to cfi, that's the recommended way to work
Thanks

Reply:
Done, changes committed.